src/main/java/org/jenkinsci/plugins/workflow/steps/WaitForConditionStep.java

```diff
@@ -56,12 +71,13 @@
          */
         private final String id = UUID.randomUUID().toString();
         private static final float RECURRENCE_PERIOD_BACKOFF = 1.2f;
-        static final long MIN_RECURRENCE_PERIOD = 250; // ¼s
-        static final long MAX_RECURRENCE_PERIOD = 15000; // ¼min
-        long recurrencePeriod = MIN_RECURRENCE_PERIOD;
+        private final long initialRecurrencePeriod;
+        long recurrencePeriod;
 
-        Execution(StepContext context) {
```

Comment:
Any binary compatibility concerns with deleting this constructor? Doesn't look like there are any callers based on a [quick search](https://github.com/search?utf8=%E2%9C%93&q=user%3Ajenkinsci+WaitForConditionStep&type=Code), but I might not be looking in the right places.

Reply:
It is package-private, so no.

Reply:
Whoops, missed that